test/asyncio_tests/test_asyncio_client.py

```diff
@@ -67,6 +68,8 @@ async def test_close(self):
     async def test_unix_socket(self):
         if env.mongod_started_with_ssl:
             raise SkipTest("Server started with SSL")
+        if os.name == "nt":
```

Comment:
Please use `sys.platform == "win32"`. Hoping we can keep using one way to detect windows across all our repros.

Reply:
ACK